hardware/index.md

```diff
@@ -220,29 +220,29 @@ Power to the micro:bit may be provided via 5V on the USB connector, or via a 3V
 
 The interface chip handles the USB connection, and is used for flashing new code to the micro:bit, sending and receiving serial data back and forth to your main computer.
 
-Due to parts availiability there are two variants of the V2.2X devices with a different interface chip; a limited run of devices were built using the nRF52833-QDAA and further devices are built using the nRF52820-QDAA.
+Due to parts availability there are two variants of the V2.2X devices with a different interface chip; a limited run of devices were built using the nRF52833-QDAA and further devices are built using the nRF52820-QDAA.
 
 As the two chip variants are of the same family, the same DAPLink firmware image runs on both chips and it provides the exact same functionality.
 
 | item          | details
 | ---           | ---
 | Model         | [nRF52833-QDAA](https://www.nordicsemi.com/products/nrf52833)
 | Core variant: | [64 MHz Arm® Cortex®-M4F](https://developer.arm.com/Processors/Cortex-M4)
-| Flash ROM     | 512KB
-| RAM           | 128KB
+| Flash ROM     | 512KB (128kB use for firmware, 128kB reserved for non-volatile storage, rest unused)
+| RAM           | 128KB (only 32 KB used)
 | Speed         | 64MHz
 | Debug capabilities | [SWD](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_nrf52%2Fstruct%2Fnrf52833.html)
-| More Info | [Software](/software/), [nRF52833 datasheet](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_nrf52%2Fstruct%2Fnrf52833.html&cp=4_1)
+| More Info | [DAPLink](/software/daplink-interface/), [nRF52833 datasheet](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fstruct_nrf52%2Fstruct%2Fnrf52833.html&cp=4_1)
```

Comment:
I think the second pipe character needs more indentation to match the rest of the table for formatting. The output looks like:

![image](https://user-images.githubusercontent.com/95928279/202161055-e4c4d4b5-dabc-4756-b0ed-1f326c3cfb53.png)

Reply:
Sorry, I'm not sure I'm following here, which pipe, this one?

```
| More Info | [DAPLink]
            ^
```

Reply:
Yes, I think so.

Reply:
But adding additional space characters in the markdown won't affect the table generated in the HTML, no?
Or at least I think it shouldn't?

Reply:
Ahh, just ignore me. It's just the text length in the right hand column that moves the column dividing line. Resolved.